### PR TITLE
Hcap 81 openshift patroni

### PR DESCRIPTION
**Pull request by ashtonmeuser**
Patroni templates
Build in `-tools` namespace

### CODE REVIEW

File: openshift/patroni.dc.yml [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+                - containerPort: 5432
+                  protocol: TCP
+              resources:
+                requests:
+                  cpu: ${CPU_REQUEST}
+                  memory: ${MEMORY_REQUEST}
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
+              terminationMessagePath: /dev/termination-log
```
Comment (fwkendall) [inline openshift/patroni.dc.yml:170]: so we do already have some logging going to file :) should we follow this path paradigm for other logging for the time being?

File: openshift/patroni.dc.yml [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                valueFrom:
+                  secretKeyRef:
+                    key: replication-username
+                    name: ${APP_NAME}-patroni
+              - name: PATRONI_REPLICATION_PASSWORD
+                valueFrom:
+                  secretKeyRef:
+                    key: replication-password
+                    name: ${APP_NAME}-patroni
+              - name: APP_USER
```
Comment (fwkendall) [inline openshift/patroni.dc.yml:123]: it seems a little weird that these are called APP_USER and APP_PASSWORD here, but seem to be the same things defined in patroni.prep where they're referred to as DB_USERNAME and DB_PASSWORD?

**Review by mauriciofreshw — DISMISSED** [review]

**Review by fwkendall — DISMISSED** [review]
Looks good, just a couple of questions from me :)

**Review by mauriciofreshw — DISMISSED** [review]

**Review by fwkendall — APPROVED** [review]

**Review by mauriciofreshw — APPROVED** [review]

File: openshift/patroni.dc.yml (hunk 1 above)
Comment (ashtonmeuser) [reply openshift/patroni.dc.yml:170]: Have not even looked into this, but it appears as if it's got some basic logging baked in. As it stands, `/dev/termination_log` is empty.

File: openshift/patroni.dc.yml (hunk 2 above)
Comment (ashtonmeuser) [reply openshift/patroni.dc.yml:123]: That does seem quite weird. However, as far as I can tell, this is to appease Patroni. So we can call it whatever we like everywhere _except_ the names of these env vars used by Patroni. I called it `DB_NAME`, `DB_USERNAME`, etc. elsewhere to align with how we've done things elsewhere.

**Pull request merged**